src/main/cluster.ts

```diff
@@ -612,7 +605,6 @@ export class Cluster implements ClusterModel, ClusterState {
       id: this.id,
       contextName: this.contextName,
       kubeConfigPath: this.kubeConfigPath,
-      workspace: this.workspace,
```

Comment:
Won't this remove the workspaces from the store file currently, can we leave this as deprecated for the time being?